tvb_epilepsy/plot/plotter.py

```diff
@@ -0,0 +1,842 @@
+import numpy
+from scipy.stats import zscore
+import matplotlib
+matplotlib.use('Qt4Agg')
+from matplotlib import pyplot, gridspec
+from matplotlib.colors import Normalize
+from mpldatacursor import HighlightingDataCursor
+from mpl_toolkits.axes_grid1 import make_axes_locatable
+from tvb_epilepsy.base.utils.log_error_utils import warning
+from tvb_epilepsy.plot.base_plotter import BasePlotter
+from tvb_epilepsy.base.model.vep.sensors import Sensors
+from tvb_epilepsy.base.utils.math_utils import compute_in_degree
+from tvb_epilepsy.base.computations.analyzers_utils import time_spectral_analysis
+from tvb_epilepsy.tvb_api.epileptor_models import EpileptorDP2D, EpileptorDPrealistic
```

Comment:
This import seems odd. We should review it when the models end up in a more final module